packages/snaps-utils/src/url.test.ts

```diff
@@ -39,8 +39,8 @@ describe('parseMetaMaskUrl', () => {
   });
 
   it('will throw on an invalid scheme', () => {
-    expect(() => parseMetaMaskUrl('metmask://client/')).toThrow(
-      'Unable to parse URL. Expected the protocol to be "metamask:", but received "metmask:".',
+    expect(() => parseMetaMaskUrl('metamask://client/')).toThrow(
```

Comment:
"metmask" is intentional here, since we're testing if the given protocol is invalid.